CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Master
 
+* Chnaged import method for Localytics (@cemaleker)
```

Comment:
Looks fine now, only this little typo needs to be fixed. Then it's ready to be merged! :)